opam

```diff
@@ -6,6 +6,6 @@ build: [
   ["ocaml" "setup.ml" "-install"]
 ]
 remove: [["ocamlfind" "remove" "inotify"]]
-depends: ["base-unix" "base-bytes" "ocamlfind"]
+depends: ["base-unix" "base-bytes" "ocamlfind" "oasis" {build}]
```

Comment:
ocamlfind's also a build dependency and the opam file version needs to be bumped to `1.2`.